src/text-field/README.md

```diff
@@ -2,7 +2,33 @@
 
 The `vscode-text-field` is a web component implementation of a [text field element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input/text).
 
-## Attributes
+![Text field hero](/docs/assets/images/text-field-hero.png)
+
+## Usage
```

Comment:
Not sure if the "input toggles" fits into this example (search, keyboard shortcuts, settings etc.) but maybe they shouldn't be in since there isn't a component for this